client/ayon_traypublisher/plugins/publish/collect_frame_data_from_folder_entity.py

```diff
@@ -2,14 +2,14 @@
 
 
 class CollectFrameDataFromAssetEntity(pyblish.api.InstancePlugin):
```

Comment:
Can we change the name too? `CollectFrameDataFromEntity`

Reply:
That'd require also to change the settings around - and hence deal with backwards compatibility there. Any examples that show how that's done?

Reply:
😞